<?xml version="1.0"?>
<launch>
    <!-- Start Arduino interface node -->
    <node pkg="ackermann_nav" type="arduino_interface" name="arduino_interface" output="screen">
        <param name="port" value="/dev/ttyUSB0" />
        <param name="baud_rate" value="115200" />
    </node>
    
    <!-- Start motor test node -->
    <node pkg="ackermann_nav" type="motor_test" name="motor_test" output="screen" />
</launch> 